### ISSUE TITLE
Masterani Download Error

### ISSUE THREAD

**Issue by miasmaejuices** [vn-ki/anime-downloader#58]
**Describe The Bug**

Unable to download from Masterani.  Error: `Expecting value: line 1 column 1 (char 0)`

**To Reproduce**
```
# anime dl 'https://www.masterani.me/anime/watch/2911-isekai-maou-to-shoukan-shoujo-no-dorei-majutsu/5' --episodes 1
anime: anime-downloader 3.3.0
anime: Extracting episode info from page
Expecting value: line 1 column 1 (char 0)
```
Tried a different episode of a new Anime to test, with the same result:
```
# anime dl 'https://www.masterani.me/anime/watch/337-suzumiya-haruhi-no-yuuutsu/1' --episodes 1
anime: anime-downloader 3.3.0
anime: Extracting episode info from page
Expecting value: line 1 column 1 (char 0)
```

**DEBUG Log**
```
# anime dl 'https://www.masterani.me/anime/watch/337-suzumiya-haruhi-no-yuuutsu/1' --episodes 1 --log-level DEBUG
INFO root: anime-downloader 3.3.0
DEBUG root: Platform: Linux-4.4.0-131-generic-x86_64-with-Ubuntu-16.04-xenial
DEBUG root: Python 3.5.2
INFO root: Extracting episode info from page
Traceback (most recent call last):
  File "/usr/local/bin/anime", line 11, in <module>
    sys.exit(cli())
  File "/usr/local/lib/python3.5/dist-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python3.5/dist-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python3.5/dist-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python3.5/dist-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/click/decorators.py", line 17, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/anime_downloader/cli.py", line 95, in dl
    anime = cls(anime_url, quality=quality)
  File "/usr/local/lib/python3.5/dist-packages/anime_downloader/sites/anime.py", line 39, in __init__
    self.get_data()
  File "/usr/local/lib/python3.5/dist-packages/anime_downloader/sites/masterani.py", line 53, in get_data
    res = res.json()
  File "/usr/local/lib/python3.5/dist-packages/requests/models.py", line 892, in json
    return complexjson.loads(self.text, **kwargs)
  File "/usr/lib/python3.5/json/__init__.py", line 319, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python3.5/json/decoder.py", line 339, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python3.5/json/decoder.py", line 357, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
```

**Comment by vn-ki** [comment]
You have to provide the anime info link, i.e. https://www.masterani.me/anime/info/337-suzumiya-haruhi-no-yuuutsu. Also it will probably not work as I have only implemented extractors for some sources of masterani.

TL;DR: Don't use masterani for now. You are better off with other providers.
